lib/action_client/base.rb

```diff
@@ -7,27 +7,39 @@ class Base < AbstractController::Base
       instance_accessor: true,
       default: ActiveSupport::OrderedOptions.new
 
+    class_attribute :submission_job,
+      instance_accessor: true,
+      default: ActionClient::SubmissionJob
+
     class << self
       def default(options)
         options.each do |key, value|
           defaults[key] = value
         end
       end
 
-      def method_missing(method_name, *args)
+      def method_missing(method_name, *arguments)
         if action_methods.include?(method_name.to_s)
-          instance = self.new(Rails.configuration.action_client.middleware)
+          instance = self.new(
+            Rails.configuration.action_client.middleware,
+            *arguments,
+          )
 
-          instance.process(method_name, *args)
+          instance.process(method_name, *arguments)
```

Comment:
Passing the arguments twice seems a little awkward. What do you think about modifying `#process` to use the `@arguments` environment variable?